From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-uffd-detect-pgtable-allocation-failures-checkpatch-fixes
Date: Wed Jan  4 05:54:55 PM PST 2023

WARNING: Statements should start on a tabstop
#127: FILE: mm/mprotect.c:353:
+		  long err = 0;						\

WARNING: Statements should start on a tabstop
#128: FILE: mm/mprotect.c:354:
+		  if (unlikely(uffd_wp_protect_file(vma, cp_flags))) {	\

WARNING: suspect code indent for conditional statements (18, 26)
#128: FILE: mm/mprotect.c:354:
+		  if (unlikely(uffd_wp_protect_file(vma, cp_flags))) {	\
+			  low##_t *p = low##_alloc(vma->vm_mm, high, addr); \

ERROR: need consistent spacing around '*' (ctx:WxV)
#129: FILE: mm/mprotect.c:355:
+			  low##_t *p = low##_alloc(vma->vm_mm, high, addr); \
 			          ^

WARNING: Statements should start on a tabstop
#130: FILE: mm/mprotect.c:356:
+			  if (p == NULL)				\

WARNING: suspect code indent for conditional statements (26, 34)
#130: FILE: mm/mprotect.c:356:
+			  if (p == NULL)				\
+				  err = -ENOMEM;			\

total: 1 errors, 5 warnings, 219 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/mm-uffd-detect-pgtable-allocation-failures.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Peter Xu <peterx@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/mprotect.c |   16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

--- a/mm/mprotect.c~mm-uffd-detect-pgtable-allocation-failures-checkpatch-fixes
+++ a/mm/mprotect.c
@@ -350,14 +350,14 @@ uffd_wp_protect_file(struct vm_area_stru
  */
 #define  change_prepare(vma, high, low, addr, cp_flags)			\
 	  ({								\
-		  long err = 0;						\
-		  if (unlikely(uffd_wp_protect_file(vma, cp_flags))) {	\
-			  low##_t *p = low##_alloc(vma->vm_mm, high, addr); \
-			  if (p == NULL)				\
-				  err = -ENOMEM;			\
-		  }							\
-		  err;							\
-	  })
+		long err = 0;						\
+		if (unlikely(uffd_wp_protect_file(vma, cp_flags))) {	\
+			low##_t *p = low##_alloc(vma->vm_mm, high, addr); \
+			if (p == NULL)					\
+				err = -ENOMEM;				\
+		}							\
+		err;							\
+	})
 
 static inline long change_pmd_range(struct mmu_gather *tlb,
 		struct vm_area_struct *vma, pud_t *pud, unsigned long addr,
_
